lib/media/video_wrapper.js

```diff
@@ -248,6 +248,11 @@ shaka.media.VideoWrapper.PlayheadMover = class {
    * @param {number} timeInSeconds
    */
   moveTo(timeInSeconds) {
+    // Skip if the time has not changed.
+    if (this.mediaElement_.currentTime == timeInSeconds) {
```

Comment:
I don't understand the need for this.  Can you help explain what is happening?

Based on this code in onTick_:

```js
    // Yay! We were successful.
    if (this.mediaElement_.currentTime != this.originTime_ ||
        this.mediaElement_.currentTime === this.targetTime_) {
      this.timer_.stop();
      return;
    }
```

It seems we would meet this condition right away.  How would we exhaust all attempts if currentTime == targetTime at the start?